__tests__/integration/one-app.spec.js

```diff
@@ -978,6 +985,13 @@ describe('Tests that require Docker setup', () => {
           });
 
           expect(result).toBe(null);
+
```

Comment:
is it worth to add a tests that deletes only the keys cached by the service worker and nothing else, as I understand it, it should keep keys in the cache that don't have anything to do with our SW